### ISSUE TITLE
make me not have to hit 15 keys in order to run

### ISSUE THREAD

**Issue by LoganDark** [chinchang/web-maker#426]
Command+Shift+5 feels like trying to hit a 15 key keybind.

Please let me change it to Command+Enter like every other app

**Comment by chinchang** [comment]
`Command+Shift+5` was chosen because it is a common shortcut in other apps too. Also, `Command+Enter` does something else - it enters a new line after the current line. 
I can may be add support for defining custom shortcuts.

**Comment by LoganDark** [comment]
Control+Enter/Alt+Enter would also work! Still don't like hitting 15 keys though

**Comment by chinchang** [comment]
Makes sense. I can add both the shortcuts (current and your suggested one) for running code.

**Comment by LoganDark** [comment]
Any progress on this? (Super sorry about my tone when I originally posted the issue, I've tried to edit my comments to make them a bit less rude)

**Comment by chinchang** [comment]
coming up in next version soon.
